Source/JavaScriptCore/runtime/JSStringInlines.h

```diff
@@ -183,6 +183,10 @@ inline void JSRopeString::resolveToBuffer(JSString* fiber0, JSString* fiber1, JS
     if (UNLIKELY(bitwise_cast<uint8_t*>(currentStackPointer()) < stackLimit))
         MUST_TAIL_CALL return JSRopeString::resolveToBufferSlow(fiber0, fiber1, fiber2, buffer, length, stackLimit);
 
+    // 0 fibers.
+    if (!fiber0)
+        return;
+
```

Comment:
Can we put this down below the 1 fiber case? This function is super hot on some tests.

Reply:
RopeString shouldn't null. So, remove `createNullRopeString` instead.